sdk/armada-protocol-common/src/common/interfaces/IArmadaManager.ts

```diff
@@ -109,6 +110,56 @@ export interface IArmadaManager {
     amount: ITokenAmount
   }): Promise<TransactionInfo[]>
 
+  /** KEEPERS TRANSACTIONS */
+
+  /**
+   * @name rebalance
+   * @description Rebalances the assets of the fleet
+   *
+   * @param rebalanceData Data of the rebalance
+   *
+   * @returns TransactionInfo The transaction information
+   */
+  rebalance(params: {
+    poolId: IArmadaPoolId
+    rebalanceData: IRebalanceData[]
+  }): Promise<TransactionInfo>
+
+  /**
+   * @name adjustBuffer
+   * @description Adjusts the buffer of the fleet
+   *
+   * @param rebalanceData Data of the rebalance
+   *
+   * @returns TransactionInfo The transaction information
+   */
+  adjustBuffer(params: {
+    poolId: IArmadaPoolId
+    rebalanceData: IRebalanceData[]
+  }): Promise<TransactionInfo>
+
+  /** GOVERNANCE TRANSACTIONS */
+
+  /**
+   * @name setFleetDepositCap
+   * @description Sets the deposit cap of the Fleet
+   *
+   * @param cap The new deposit cap
+   *
+   * @returns TransactionInfo The transaction information
+   */
+  setFleetDepositCap(params: { poolId: IArmadaPoolId; cap: ITokenAmount }): Promise<TransactionInfo>
```

Comment:
there's bunch of the governance methods missing. is that on purpose?

```
setFleetDepositCap
setTipJar
setTipRate
addArk
addArks
removeArk
setArkDepositCap
setArkMaxRebalanceOutflow
setArkMaxRebalanceInflow
setMinimumBufferBalance
updateRebalanceCooldown
forceRebalance
emergencyShutdown
```

Reply:
Oh! I think I misread the contract, the `setFleeDepositCap` and `setTipJar` functions have no comments and then `setTipRate` does have a comment and I thought it was a different section of it. I'll add all those